docker-compose.yml

```diff
@@ -17,10 +17,12 @@ services:
     ports:
       - 6379:6379
     command: redis-server --appendonly yes
+    depends_on:
+      - postgres
```

Comment:
redis does not depend on postgres

Reply:
Thank you for the remark! I admit that I didn’t consider that Redis does not depend on Postgres. My goal was to ensure that all container processes started after the database was successfully launched. 😊